mantidproject/mantid
# EQSANS I(Q,lambda)

Instrument team wanted to implement an alternate way of computing I(Q) from the I(Q,wavelength) curves for each wavelength bin. For this, we need to:

- Add a workspace group with the I(Q,wavelength) to the azimuthal averaging code.
- Add a n algorithm to determine the scaling factor of each I(Q,wavelength) and assemble the final I(Q).

**To test:**

- Inspect the code
- Make sure all the existing system tests pass. They should exercise all the existing functionality.
- Run the example code below. The number of wedge workspaces should be 3 for each frame. One run will produce one I(Q), and the second run will produce one I(Q) for each of the two frames.

There is not "issue" for this PR, all the info is here.

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
